src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -12,6 +12,9 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage)</TargetsForTfmSpecificContentInPackage>
+    <!--Partial elements should declare access modifier: Disabled for this project to simplify handling in common project. See also there.
+    If we disable this here, the warning is still emmited for the individual projects.-->
+    <NoWarn>$(NoWarn);SA1205</NoWarn>
```

Comment:
I would need to disable it in many files in the common project, which would somewhat jeopardize the nice solution with not specifying a visibility for the internal classes. 

As the documentation says: Because all files are also compiled using a separate project, the warning does come up for classes not in the common project.